src/bin/tikv-server.rs

```diff
@@ -350,7 +363,7 @@ fn overwrite_config_with_cmd_args(config: &mut TiKvConfig, matches: &ArgMatches)
         config.raft_store.capacity = capacity;
     }
 
-    if matches.is_present("import-mode") {
+    if matches.is_present("import-mode") || matches.is_present("import-server") {
```

Comment:
what is the different between import-mode or import-server?

Reply:
import-mode tunes some configs automatically, while import-server runs as an import server, which has nothing to do with tikv. Don't pay too much attention here now, I will try to move this to other place and build import server as a standalone binary in next pr.